apps/store/src/pages/[...slug].tsx

```diff
@@ -33,10 +33,12 @@ export const getStaticProps: GetStaticProps<StoryblokPageProps, StoryblokQueryPa
 }) => {
   const slug = params?.slug ? params.slug.join('/') : 'home'
   const story = await getStoryBySlug(slug, preview)
+  const globalStory = await getGlobalStory(preview)
```

Comment:
nit: this could be run in parallel

```suggestion
const [story, globalStory] = await Promise.all([
  getStoryBySlug(slug, preview)
  getGlobalStory(preview)
])
```